src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -405,21 +427,92 @@ void registerBlobs(List<BlobMetadata> blobs) {
                                 .forEach(
                                     channelStatus -> {
                                       if (channelStatus.getStatusCode() != RESPONSE_SUCCESS) {
-                                        logger.logWarn(
-                                            "Channel has been invalidated because of failure"
-                                                + " response, name={}, channel sequencer={},"
-                                                + " status code={}",
-                                            channelStatus.getChannelName(),
-                                            channelStatus.getChannelSequencer(),
-                                            channelStatus.getStatusCode());
-                                        channelCache.invalidateChannelIfSequencersMatch(
-                                            chunkStatus.getDBName(),
-                                            chunkStatus.getSchemaName(),
-                                            chunkStatus.getTableName(),
-                                            channelStatus.getChannelName(),
-                                            channelStatus.getChannelSequencer());
+                                        // If the chunk queue is full, we wait and retry the chunks
+                                        if (channelStatus.getStatusCode()
+                                                == RESPONSE_ERR_ENQUEUE_TABLE_CHUNK_QUEUE_FULL
+                                            && executionCount < MAX_API_RETRY) {
```

Comment:
We have a constant `MAX_RETRIES` for general HTTP retries, could you make it clear in the name if this is only for Streaming Ingest APIs?